tests/e2e/middleware/api-metrics.middleware.spec.ts

```diff
@@ -83,6 +83,46 @@ describe('Api Metrics Middleware', () => {
     expect(metricService.getHistogram).toHaveBeenCalledTimes(3);
   });
 
+  it('registers api request and response metrics with dynamic attributes when has callback', async () => {
+    const testingModule = await Test.createTestingModule({
+      imports: [
+        OpenTelemetryModule.forRoot({
+          metrics: {
+            apiMetrics: {
+              enable: true,
+              dynamicAttributes: () => ({ dynamicLabel: 'dynamicLabelValue' }),
```

Comment:
Could you make sure you use something from req here, this PR was introduced to support dynamic attributes but the test is testing the same as defaultAttributes (static values)